app/js/components/orderEntry/PastOrder.jsx

```diff
@@ -1,31 +1,61 @@
 import React from 'react';
 import { format } from 'date-fns';
 
-export const PastOrder = (props) => {
-  const {
-    drug, auditInfo, dose, doseUnits, frequency,
-    route, duration, durationUnits, dosingInstructions,
-    quantityUnits, quantity,
-  } = props;
-  return (
-    <tr>
-      <td>
-        {format(auditInfo.dateCreated, 'DD/MM/YYYY | HH:MM:SS')}
-      </td>
-      <td>Inactive</td>
-      <td>
-        {drug.display}:
-        {dose ? ` ${dose}` : ""}
-        {doseUnits.display ? ` ${doseUnits.display}` : ""}
-        {frequency.display ? `, ${frequency.display}` : ""}
-        {route.display ? `, ${route.display}` : ""}
-        {duration ? `, for ${duration}` : ""}
-        {durationUnits.display ? ` ${durationUnits.display} total` : ""}
-        {dosingInstructions ? `, (${dosingInstructions})` : ""}
-        {(dosingInstructions && quantityUnits.display) ? `, (Dispense: ${quantity} ${quantityUnits.display})` : ""}
-      </td>
-      <td>&nbsp;&nbsp;<span className="icon-ban-circle" /></td>
-    </tr>
-  );
-};
+export class PastOrder extends React.Component {
+  render() {
+    const {
+      drug, dosingType, action, auditInfo, dose, doseUnits, frequency,
+      route, duration, durationUnits, dosingInstructions,
+      quantityUnits, autoExpireDate, quantity,
+    } = this.props;
+
+    const getStatus = () => {
+      if (action === "NEW" && autoExpireDate !== null) {
+        return 'Completed';
+      } else if (autoExpireDate !== null && action === "REVISE") {
+        return 'Revised';
+      } else if (autoExpireDate === null) {
+        return 'Discontinued';
+      }
+    };
+
+    let details;
+    if (dosingType === 'org.openmrs.SimpleDosingInstructions') {
+      details = (
+        <p>
+          {drug.display}:
```

Comment:
You should have commas between this values

Reply:
Not all of the need commas because they are being displayed as a statement or sentence